### PR TITLE
[IA-5008] Update Hail to 0.2.131

### PR DESCRIPTION
**Pull request by rtitle**
https://broadworkbench.atlassian.net/browse/IA-5008

Depends on Dataproc upgrade: https://github.com/DataBiosphere/leonardo/pull/4699

### CODE REVIEW

**Comment by rtitle** [comment]
The GHA failed, investigating why..

File: .github/workflows/test-terra-jupyter-aou.yml [hunk 1]
```diff
@@ -84,7 +84,7 @@ jobs:
           --workdir=/tests \
           --entrypoint="" \
           terra-jupyter-aou:smoke-test \
-          /bin/bash -c 'for nb in {terra-jupyter-python/tests,terra-jupyter-aou/tests}/*ipynb ; do jupyter nbconvert --to html --ExecutePreprocessor.allow_errors=True --execute "${nb}" ; done'
+          /bin/bash -c 'for nb in {terra-jupyter-python/tests,terra-jupyter-aou/tests}/*ipynb ; do jupyter nbconvert --to html_embed --ExecutePreprocessor.allow_errors=True --execute "${nb}" ; done'
```
Comment (rtitle) [inline .github/workflows/test-terra-jupyter-aou.yml:87]: This was failing with:
```
nbconvert.exporters.base.ExporterNameError: Unknown exporter "html", did you mean one of: html_ch, html_embed, html_toc, selectLanguage?
```

I changed all instances of `--to hrml` to `--to html_embed`

File: .github/workflows/test-terra-jupyter-python.yml [hunk 2]
```diff
@@ -112,7 +112,7 @@ jobs:
           --workdir=/tests \
           --entrypoint="" \
           terra-jupyter-python:smoke-test \
-          /bin/sh -c "pip3 install pytest ; pytest"
+          /bin/sh -c "pip3 install pytest --user ; pytest"
```
Comment (rtitle) [inline .github/workflows/test-terra-jupyter-python.yml:115]: Fix error:
```
ERROR: Could not install packages due to an OSError: [Errno 13] Permission denied: 'top_level.txt'
Consider using the `--user` option or check the permissions.

/bin/sh: 1: pytest: not found
```

**Comment by rtitle** [comment]
I tried to fix some issues with the automated tests, still iterating. It's slow because the images are so huge. 😅  Requesting reviews anyway.

@lucymcnatt is updating `conf.json` all that is needed -- will it automatically build and publish the new `terra-jupyter-hail` image? (I seem to remember that is the case)

**Review by lucymcnatt — APPROVED** [review]

**Review by cindy-broadinstitute — APPROVED** [review]

**Comment by rtitle** [comment]
All tests passed! Merging this.

**Pull request merged**